modules/book-list.js

```diff
@@ -0,0 +1,30 @@
+import DomManipulator from './dom-manipulator.js';
+import Storage from './Storage.js';
+
+class BookList {
+  constructor() {
+    this.books = Storage.get();
+    this.domManipulator = new DomManipulator(this.removeBook.bind(this));
+    this.books.forEach((book) => this.domManipulator.insertBookToDom(book));
+  }
+
+  addBook(newBook) {
+    // Insert new book to dom
+    this.domManipulator.insertBookToDom(newBook);
+
+    // Update book list with new book
+    this.books.push(newBook);
+    Storage.set(this.books);
+  }
+
+  removeBook(id) {
+    // Remove from the book array
+    const parseId = parseInt(id, 10);
+    const index = this.books.findIndex((book) => book.id === parseId);
+    this.books.splice(index, 1);
+    // Remove from local storage
+    Storage.set(this.books);
+  }
+}
+
```

Comment:
- [ ] Hi you really did an amazing job but as part of the requirements, you are supposed to use ES6 function components instead of classes. Please kindly change all your classes to ES6 arrow functions

Reply:
* As mentioned by the previous CR you are required to use arrow functions [Check requirement here](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Refactor%20your%20functions%20to%20arrow%20functions.)
![image](https://user-images.githubusercontent.com/52497246/220153632-ddb8feae-0e89-48ed-85fc-72effc683097.png)